analytics/CAR-2021-01-006.yaml

```diff
@@ -0,0 +1,32 @@
+---
+title: Unusual Child Process spawned using DDE exploit
+submission_date: 2020/12/03
+information_domain: 'Host'
+platforms:
+  - Windows
+subtypes:
+  - Process
+analytic_types:
+  - TTP
+contributors:
+  - Cyware Labs
+id: CAR-2021-01-006
+description: |-
+  Adversaries may use Windows Dynamic Data Exchange (DDE) to execute arbitrary commands. DDE is a client-server protocol for one-time and/or continuous inter-process communication (IPC) between             applications. Once a link is established, applications can autonomously exchange transactions consisting of strings, warm data links (notifications when a data item changes), hot data links               (duplications of changes to a data item), and requests for command execution.
+  - technique: T1559
+    coverage: Low
+    subtechniques:
+      - T1559.002
+    tactics:
+      - TA0002
+implementations:
+- name: Splunk search - Unusual Child Process spawned using DDE exploit
+
+  description:  This Splunk query looks for any executable invocations from an Excel file.
+
+  code: |-
+    index = * ParentImage="*excel.exe*" Image="*.exe" | table _raw, ParentImage, Image
```

Comment:
This analytic purports to be looking for arbitrary commands via DDE, but the splunk query is simply looking for instances of excel.exe. My guess is that this was a mistake; please fix and modify the PR. Thanks.

Reply:
Hello @pcmcpherson we are looking for any exe spawns in this in the Image parameter with the Parent Image being Excel.exe. Let me know if anything needs to be changed here